rules/macos/defense_evasion_privacy_controls_tcc_database_modification.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2020/12/23"
+maturity = "production"
+updated_date = "2020/12/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of sqlite3 to directly modify the TCC SQLite database. This may indicate an attempt to bypass macOS
+privacy controls, including access to sensitive resources like the system camera, microphone, address book, and
+calendar.
+"""
```

Comment:
```suggestion
description = """
Identifies the use of sqlite3 to directly modify the Transparency, Consent, and Control (TCC) SQLite database. This may
indicate an attempt to bypass macOS privacy controls, including access to sensitive resources like the system camera,
microphone, address book, and calendar.
"""
```